mayhem/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM fuzzers/go-fuzz:1.2.0
```

Comment:
Any reason we update go here? Setting an entrypoint isn't strictly necessary. You can remove the commented commands.

Reply:
go is updated to the most recent version (1.20.3) because fuzzers/go-fuzz:1.2.0 uses go version 1.13; the file `enola/enola.go` imports `_ "embed"`, but the `_` is not supported by go 1.13- Unfortunately, what im trying to fuzz are functions of a struct contained within this file, so updating go within the docker image seemed the easiest way around this